src/Components/Favorites/Favorites.js

```diff
@@ -1,15 +1,29 @@
+import { useEffect, useState } from "react";
 
 const Favorites = () => {
+    const [allFavorites, setAllFavorites] = useState([])
+    useEffect(() => {
+    setAllFavorites(JSON.parse(localStorage.getItem('favorites')))
+    }, [])
+
+const displayFavorites = () => {
+return allFavorites.map(favorite => {
     return (
-        <section className="activity-card">
-    <div></div>
+    <div>
     <ul>
-        <li>Activity:</li>
-        <li>Category:</li>
-        <li>Accessibility:</li>
-        <li>Price:</li>
-        <li>Participants:</li>
+        <li>Activity:{favorite.activity}</li>
+        <li>Category:{favorite.type}</li>
+        <li>Accessibility:{favorite.accessibility}</li>
+        <li>Participants:{favorite.participants}</li>
     </ul>
+    </div>
+    )
+    })
+}
+
+    return (
+        <section className="activity-card">
+        {displayFavorites()}
```

Comment:
I was also thinking about making a separate component for favorites where I made new favorites cards like this. I think this is a smart way to go about it because you can control the favorited cards to be exactly how you want.